Makefile

```diff
@@ -29,17 +29,33 @@ image_base="ubuntu"
 image_tag="20.04"
 endif
 
+buildx=0
```

Comment:
@qsliu2017 Hi, I was wondering that how should we use this `buildx`? Do we need to pass `buildx=1` when running `make`?

Reply:
@imjoey Yes, I want to make it optional and do not influent current use, so passing any value other than 0 can active buildx

Reply:
@qsliu2017 OK. We would better provide the usage of this new `parameter` in README.md, what do you think?